app/src/main/java/com/example/paulsuarez/a26_simple_android_apps_simple_counter/MainActivity.java

```diff
@@ -0,0 +1,208 @@
+            public void onClick(View view) {
+                String text = mEditText.toString();
+                String text2 = mEditText.getText().toString();
+
+                totalClicks = Integer.parseInt(text2);
+                mClicks.setText("clicks: " + totalClicks);
+            }
+        });
+
+        mClickMe.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                totalClicks++;
+                mClicks.setText("clicks: " + totalClicks);
+
+            }
+        });
+
+        mAButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                aClicks++;
+                mAText.setText("Bitcoins Ordered: " + aClicks);
+
+            }
+        });
+
+
+        mBButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                bClicks++;
+                mBText.setText("Litecoins Ordered: " + bClicks);
+
+            }
+        });
+
+
+        mCButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                cClicks++;
+                mCText.setText("EncryptNodes Ordered: " + cClicks);
+
+            }
+        });
+
+
+
+
+        // testing
+        currentIndex = 0;
+
+        picture = findViewById(R.id.picture);
+        name = findViewById(R.id.name);
+        desc = findViewById(R.id.description);
+
+        prev = findViewById(R.id.prev);
+        next = findViewById(R.id.next);
+
+        showCoin();
+
+        prev.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                decrement();
+            }
+        });
+        next.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                increment();
+            }
+        });
+        //
+
+
+
+    }
+
+
+
+
+    // trying to make a single scrollable app:
+
+    private int currentIndex;
+
+    CryptoMenu[] coins = {
+            new CryptoMenu("Bitcoin", R.drawable.bitcoin, "Bitcoin uses peer-to-peer technology to operate with no central authority or banks; managing transactions and the issuing of bitcoins is carried out collectively by the network."),
+            new CryptoMenu("Litecoin", R.drawable.litecoin, "Litecoin is a peer-to-peer Internet currency that enables instant, near-zero cost payments to anyone in the world."),
+            new CryptoMenu("EncryptNode", R.drawable.encrytnode, "EncryptNode aims to simplify blockchain technology and make it accessible to everyone. Hashed using a SHA256 encryption algorithm; EncryptNode is both efficient and secure!")
+    };
+
+    private ImageView picture;
+    private TextView name;
+    private TextView desc;
+
+    private Button prev;
+    private Button next;
+
+
+
+
```

Comment:
Since the below methods look like implementation details, consider making them private. Generally a good approach is to make everything private or protected and only open up things as public if there is a specific need.